app/components/elements/forms/select_day_field_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Elements
+  module Forms
+    # Component for form day of the month fields
+    class SelectDayFieldComponent < Forms::FieldComponent
+      def value
+        form.object.public_send(field_name)
+      end
+
+      def prefix
+        form.object_name
+      end
+
+      def classes
+        merge_classes('form-select', form.object.errors[field_name].present? ? 'is-invalid' : nil)
+      end
```

Comment:
It looks like currently on `main`, all the `is-invalid` jazz is handled by the invalid feedback component and the tab error controller. Can you say more about why the current invalidity infrastructure doesn't/shouldn't handle this?

Reply:
Yes, because `select_day` isn't bound to a form builder instance. I've added a comment to the top of the class.